cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -200,6 +200,14 @@
     </description>
   </property>
 
+  <property>
+    <name>zookeeper.client.timeout.millis</name>
```

Comment:
think it would be better to name this something like 'zookeeper.client.startup.timeout.millis' or something like that. Otherwise if I look at just the name and not the description, I would probably think it means something else.